tests/integration/components/sl-calendar-test.js

```diff
@@ -0,0 +1,1461 @@
+        }}
+        {{sl-calendar
+            action="testAction2"
+            content=content
+            currentYear=currentYear2
+            currentMonth=currentMonth2
+        }}
+    ` );
+
+    this.on( 'testAction1', () => {
+        assert.ok(
+            true,
+            'The test action 1 was called'
+        );
+    });
+
+    this.on( 'testAction2', () => {
+        assert.ok(
+            true,
+            'The test action 2 was called'
+        );
+    });
+
+    this.$( '>:nth-child(2) .active' ).click();
+});
+
+test( 'Dual instance: Navigating Forward by Month', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear1', currentYear );
+    this.set( 'currentYear2', currentYear );
+
+    const currentMonth = 9;
+    this.set( 'currentMonth1', currentMonth );
+    this.set( 'currentMonth2', currentMonth );
+
+    this.render( hbs`
+        {{sl-calendar
+            currentMonth=currentMonth1
+            currentYear=currentYear1
+        }}
+        {{sl-calendar
+            currentMonth=currentMonth2
+            currentYear=currentYear2
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance one: current month is set correctly'
+    );
+
+    assert.equal(
+        this.$( '>:nth-child(2) .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance two: current month is set correctly'
+    );
+
+    this.$( '>:nth-child(2) .next' ).click();
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance one: current month has not changed'
+    );
+
+    assert.equal(
+        this.$( '>:nth-child(2) .datepicker-switch' ).text().trim(),
+        'October ' + currentYear,
+        'Component instance two: Current month advances one month'
+    );
+});
+
+test( 'Dual instance: Navigating Backward by Month', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear1', currentYear );
+    this.set( 'currentYear2', currentYear );
+
+    const currentMonth = 9;
+    this.set( 'currentMonth1', currentMonth );
+    this.set( 'currentMonth2', currentMonth );
+
+    this.render( hbs`
+        {{sl-calendar
+            currentMonth=currentMonth1
+            currentYear=currentYear1
+        }}
+        {{sl-calendar
+            currentMonth=currentMonth2
+            currentYear=currentYear2
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance one: current month is set correctly'
+    );
+
+    assert.equal(
+        this.$( '>:nth-child(2) .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance two: current month is set correctly'
+    );
+
```

Comment:
Are either of the previous two assertions needed? Other tests in this suite have ensured this functionality, and since they year and month values are not being set to different ones it's not like we're checking for any bleed out.